docs/content/developer/cryptography/transaction-auth/offline-signing.mdx

```diff
@@ -82,15 +82,15 @@ Mutated Objects:
 Alternatively, you can use the active key in IOTA Keystore to sign and output a Base64-encoded sender signed data with flag `--serialize-signed-transaction`. 
 
 ```shell
-iota client transfer-iota --to <IOTA-ADDRESS> --iota-coin-object-id <COIN-OBJECT-ID> --serialize-signed-transaction
+iota client pay-all-iota --input-coins <COIN-OBJECT-ID> --recipient <IOTA-ADDRESS> --gas-budget 2000000 --serialize-unsigned-transaction
```

Comment:
Just making sure the signed to unsigned isn't a copy-paste error

Reply:
It is, good catch

Reply:
Fixed in https://github.com/iotaledger/iota/pull/4701/commits/47296113c46b94f38c9bc8ad1a9d233fae8c160b